core/sail/nativerdf/src/test/java/org/eclipse/rdf4j/sail/nativerdf/NativeStoreTest.java

```diff
@@ -43,7 +43,7 @@
 	@Override
 	protected NotifyingSail createSail() throws SailException {
 		try {
-			NotifyingSail sail = new NativeStore(tempDir.newFolder("nativestore"), "spoc,posc");
+			NotifyingSail sail = new NativeStore(tempDir.newFolder(), "spoc,posc");
```

Comment:
I'm not sure why you changed this, but this change causes a test failure (because the test assumes there is a 'nativestore' subdir)